common/lib/capa/capa/templates/optioninput.html

```diff
@@ -1,7 +1,7 @@
 <% doinline = "inline" if inline else "" %>
 
 <form class="inputtype option-input ${doinline}">
-  <label class="problem-group-label" for="input_${id}">
+  <label class="problem-group-label" for="input_${id}" id="${response_data['label']}">
```

Comment:
I'm very confused about how the ID can also be the user-visible text. Can you provide an example to a problem of this type on your sandbox?